draft/2020-06-09-this-week-in-rust.md

```diff
@@ -81,38 +81,35 @@ If you are a Rust project owner and are looking for contributors, please submit
-* [`impl Step for char` (make `Range*<char>` iterable)](https://github.com/rust-lang/rust/pull/72413)
-* [add `Peekable::next_if`](https://github.com/rust-lang/rust/pull/72310)
-* [various minor improvements to `Ipv6Addr::Display`](https://github.com/rust-lang/rust/pull/72407)
-* [`SocketAddr` and friends now correctly pad its content](https://github.com/rust-lang/rust/pull/72398)
-* [implement PartialOrd and Ord for SocketAddr*](https://github.com/rust-lang/rust/pull/72239)
-* [tweak and stabilize `Atomic`N`::fetch_update`](https://github.com/rust-lang/rust/pull/71843)
-* [stabilize `Atomic`N`::fetch_`{`min`, `max`}](https://github.com/rust-lang/rust/pull/72324)
-* [stdarch: add 64 bit integer AVX512f comparisons and the intrinsics needed to test them](https://github.com/rust-lang/stdarch/pull/856)
-* [stdarch: add 64 bit AVX512f le and ge comparisons](https://github.com/rust-lang/stdarch/pull/861)
-* [libm: use macros for more division/array checks](https://github.com/rust-lang/libm/pull/244)
+* [be more careful around `ty::Error` in generators](https://github.com/rust-lang/rust/pull/72764)
+* [fulfill: try using `SmallVec` or `Box` for `stalled_on`](https://github.com/rust-lang/rust/pull/72776)
+* [`impl AsRef<[T]> for vec::IntoIter<T>`](https://github.com/rust-lang/rust/pull/72583)
+* [chalk: get ready for the first publish](https://github.com/rust-lang/chalk/pull/483)
+* [free `default()` forwarding to `Default::default()`](https://github.com/rust-lang/rust/pull/73001)
+* [stabilize `std::io::Buf{Reader, Writer}::capacity`](https://github.com/rust-lang/rust/pull/72924)
+* [add associated consts `MIN`/`MAX` for `Wrapping<Int>`](https://github.com/rust-lang/rust/pull/72891)
+* [de-promote Duration::from_secs](https://github.com/rust-lang/rust/pull/71796)
+* [compiler-builtins: manually patch ret instruction for LVI](https://github.com/rust-lang/compiler-builtins/pull/359)
+* [cargo: add environment variables to identify the binary and crate name](https://github.com/rust-lang/cargo/pull/8270)
+* [cargo: allow Windows dylibs without dll suffix](https://github.com/rust-lang/cargo/pull/8310)
+* [cargo: better error message when passing in relative path to `Workspace::new`](https://github.com/rust-lang/cargo/pull/8321)
+* [cargo: don't hash executable filenames on apple platforms](https://github.com/rust-lang/cargo/pull/8329)
+* [cargo: support `{prefix}` and `{lowerprefix}` markers in `config.json` `dl` key](https://github.com/rust-lang/cargo/pull/8267)
+* [cargo: warn if using hash in git URL](https://github.com/rust-lang/cargo/pull/8297)
+* [cargo: reset lockfile information between resolutions](https://github.com/rust-lang/cargo/pull/8274)
+* [crates.io: fix issue where crates.io allowed the plus sign in crate names](https://github.com/rust-lang/crates.io/pull/2551)
+* [docs.rs: print a backtrace for crates which fail to build](https://github.com/rust-lang/docs.rs/pull/823)
```

Comment:
Thanks for considering docs.rs for TWIR! I'm not sure though that this was a great candidate for notable changes, though - it doesn't affect anything user-facing, only internally logging for bugs in docs.rs (things like https://github.com/rust-lang/docs.rs/issues/797). A better choice might have been https://github.com/rust-lang/docs.rs/pull/807, which fixed a UI bug, or https://github.com/rust-lang/docs.rs/pull/802, which no longer returns 404 on internal errors.

Of course TWIR has already been  published for this week so no big deal.